.github/workflows/update_version.yml

```diff
@@ -46,5 +46,4 @@ jobs:
           --prj_ver ${{ github.event.inputs.project_version }} \
           --gh_token "${GH_TOKEN}" \
           --pipeline \
-          --exec_path "$(pwd)" \
-          --postgres_version "${POSTGRES_VERSION}"
+          --exec_path "$(pwd)" 
```

Comment:
Removed postgres version parameter wgich causes error since this parameter had removed from tools script

Reply:
could you remind me how do we decide postgres version now ?

Reply:
I update them manually since 12,13 and 14 versions are different. Previously, there was only one version since such parameter was needed